src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Metrics/MetricsEventSource.cs

```diff
@@ -0,0 +1,356 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Diagnostics.Tracing;
+using System.Globalization;
+using System.Runtime.Versioning;
+using System.Text;
+
+namespace System.Diagnostics.Metrics
+{
+    [EventSource(Name = "System.Diagnostics.Metrics")]
+    internal class MetricsEventSource : EventSource
```

Comment:
>MetricsEventSource [](http://example.com/codeflow?start=19&length=18)

could you please add some comment here regarding the spec syntax and listing some examples? I didn't review all files yet so far I didn't see if you already have it somewhere else.

Reply:
Nope you are accurate, it hadn't been documented in the comments yet : )